thirdparty/google-styleguide/run_linter.py

```diff
@@ -1,6 +1,10 @@
 import argparse
+import multiprocessing
 import os
 import cpplint
+from multiprocessing import Pool
```

Comment:
Only one of line 2 or line 5 should be enough.

Reply:
Done.